airflow/kubernetes/pod_generator.py

```diff
@@ -292,6 +299,15 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals
         # Attach sidecar
         self.extract_xcom = extract_xcom
 
+    def qualify_image_url(self, image: str) -> str:
```

Comment:
Would like to update this method name to be more specific to xcom-sidecar

Reply:
Maybe it's simpler to add a parameter for the side car image full path instead of using a docker registry? This is specific but will match to your comments. Is it ok for you?

Reply:
absolutely!